atlasdb-dbkvs/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/OracleTableNameUnmapperTest.java

```diff
@@ -184,40 +187,54 @@ public void getLongTableNamesFromMappingTableDoesMultipleQueriesIfMoreTablesThan
     }
 
     @Test
-    public void getLongTableNamesThrowsIfPartialResultsReturned() {
-        mockShortNamesToLongNamesQuery(Set.of("test"));
+    public void getShortToLongTableNamesFromMappingTableCanReturnPartialResults() {
+        Map<String, String> shortNamesToLongNames = Map.of("short_test", "long_test");
+        mockShortNamesToLongNamesQuery(shortNamesToLongNames);
 
-        assertThatThrownBy(() -> oracleTableNameUnmapper.getLongTableNamesFromMappingTable(
-                        connectionSupplier, Set.of("test", "test_2")))
-                .isInstanceOf(TableMappingNotFoundException.class)
-                .hasMessageContaining("Some of the tables")
-                .hasMessageContaining("do not have a mapping. This might be because these tables do not exist.");
+        Map<String, String> mappedNames = oracleTableNameUnmapper.getShortToLongTableNamesFromMappingTable(
+                connectionSupplier, Set.of("short_test", "test_2"));
+        assertThat(mappedNames).isEqualTo(shortNamesToLongNames);
     }
 
     @Test
-    public void getLongTableNamesThrowsIfTooManyResultsReturned() {
-        mockShortNamesToLongNamesQuery(Set.of("test", "test_2"));
+    public void getShortToLongTableNamesFromMappingTableThrowsIfTooManyResultsReturned() {
+        Map<String, String> nameMapping = Map.of("short_test", "long_test", "short_test_2", "long_test_2");
```

Comment:
eh, I guess this should be `short_tesT` or something like that because of the lower check

Reply:
(discussed offline: this is also a valid way of modelling a misbehaving query!)